examples/core/FixedWing_Demo.py

```diff
@@ -0,0 +1,109 @@
+"""Spawn a single fixed wing UAV on x=0, y=0, z=50, with 0 rpy."""
```

Comment:
For a demo, I think remove the controller usage, ideally should be programmatic only.

Reply:
Removed this file as per [Comment](https://github.com/jjshoots/PyFlyt/pull/2/files/5571be8232e935e61bc14313c63786b3c1691296#r1065014861)